src/client/styling/form.css

```diff
@@ -0,0 +1,97 @@
+/* FORM PAGE FORMATTING */
```

Comment:
Nice. I was going to suggest having separate files for styling so it stays manageable as the code expands.